modules/changes-logs/classes/class-changes-handle-wp-comments.php

```
@@ -0,0 +1,201 @@
+    public static function callback_change_comment_deleted( $comment_id ) {
+        self::change_comment_action( $comment_id, 1560 );
+    }
+
+    /**
+     * Fires immediately after a comment is inserted into the database.
+     *
+     * @param int        $comment_id       The comment ID.
+     * @param int|string $comment_approved 1 if the comment is approved, 0 if not, 'spam' if spam.
+     * @param array      $comment_data     Comment data.
+     */
+    public static function callback_change_comment( $comment_id, $comment_approved, $comment_data ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$comment_approved'. (Unused Code Rules)

(UnusedFormalParameter)